src/styles/partials/components/_card-container.scss

```diff
@@ -9,10 +9,14 @@
   padding: $padding-large;
   text-align: center;
   width: 100%;
-
-  .dg_icon-container {
-    color: $gold;
-    font-size: $largest-icon-size;
-    margin-bottom: 20px;
+}
+.dg_icon-container {
+  color: $gold;
+  font-size: $largest-icon-size;
+  margin-bottom: 20px;
+}
+.dg_card__content {
```

Comment:
was this added, if so why? I didn't see anything about this in the bug?

Reply:
I believe it was from previous PR fix "change the font size to 18px". I must have created branch from that fix.

Reply:
no problem, i was hitting these backwards, we can wait until that one is completed